Build/NuGet/package-data.xml

```diff
@@ -25,8 +25,8 @@
   </commonProperties>
   <packages>
     <package id="Microsoft.ChakraCore" nuspecFile="Windows.DotNet.All\Primary.nuspec" />
-    <package id="Microsoft.ChakraCore.Symbols" nuspecFile="Windows.DotNet.All\Symbols.nuspec" />
```

Comment:
Is 'symbols` the same for all platforms or does it need to match other packages?

Reply:
Each platform has its own symbols. One of the next steps will create “symbol” packages for each platform (see the fourth step of issue #6586).